src/app/components/main-sections/contact-us/index.tsx

```diff
@@ -4,17 +4,19 @@ import { sunAndMountains } from "@/app/utils/spline-scenes"
 
 export default function ContactUs() {
   return (
-    <section className="h-screen relative">
-      <SplineViewer scene={sunAndMountains} className="absolute top-0 left-0 w-full h-full -z-10" />
-      <div className="relative flex flex-row justify-around h-full items-center">
-        <div className="flex flex-col w-6/12 border-r-[0.1rem] border-[#bcbcbca9] h-5/6 justify-center items-center">
-          <div className="flex flex-col gap-8">
-            <h1 className="max-w-[340px] text-7xl">Tell us your new challenge</h1>
-            <p className="text-xl text-[#c4c4c4b1]">We are excited to hear from you.</p>
+    <section className="h-screen w-screen">
+      <SplineViewer scene={sunAndMountains} className="absolute -z-10" />
+      <div className="flex flex-col w-full h-screen xl:flex-row xl:justify-around xl:items-center justify-around">
+        <div className="flex flex-col w-full xl:w-6/12 xl:border-r-[0.1rem] border-[#bcbcbca9] xl:h-5/6 justify-center items-center">
+          <div className="flex flex-col gap-8 max-w-[340px]">
+            <h1 className="text-7xl text-center xl:text-start">Tell us your new challenge</h1>
+            <p className="text-xl text-center xl:text-start">We will contact you to discover your incredible project.</p>
           </div>
         </div>
-        <div className="w-6/12 h-full flex items-center justify-center px-12">
-          <ContactUsForm />
+        <div className="w-full xl:w-6/12 flex items-center justify-center px-12">
+          <div className={"w-full md:w-10/12 2xl:w-6/12"}>
```

Comment:
some of them are on quotes and other classNames are with braces?

Reply:
also two divs are not correct, we could resolve it with just one of them, one container and the item.